### ISSUE TITLE
Bookmark button next to title

### ISSUE THREAD

**Issue by dnoneill** [sul-dlss/earthworks#1198]
niqui has done this work in BL, needs a BL release and GeoBL release. Note mobile doesn't have the word "bookmark", only the icon.

**Comment by dnoneill** [comment]
blocked by needing a new blacklight release. I want to wait until https://github.com/projectblacklight/blacklight/pull/3277 is merged before asking for a new release, otherwise the css on that branch won't get folded into the release.

There is a draft PR to add Bookmark to search results page. It just displays without the icon on this branch. https://github.com/sul-dlss/earthworks/pull/1199
